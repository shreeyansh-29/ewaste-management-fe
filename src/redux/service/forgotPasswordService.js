import {FORGOT_PASSWORD} from "../../container/constant/constant";
import api from "../../core/utilities/httpProvider";

export const helper = (data) => {
//   console.log("email", data.email.email);
  const email = data.email.email;
  return api.post(FORGOT_PASSWORD, {
    email: email,
  });
};
