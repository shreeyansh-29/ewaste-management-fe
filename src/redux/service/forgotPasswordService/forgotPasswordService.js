import {FORGOT_PASSWORD} from "../../../container/constant/constant";
import api from "../../../core/utilities/httpProvider";

export const helper = (data) => {
  const email = data.email.email;
  return api.post(FORGOT_PASSWORD, {
    email: email,
  });
};
