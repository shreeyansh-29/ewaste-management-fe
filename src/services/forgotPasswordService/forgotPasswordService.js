import {FORGOT_PASSWORD} from "../../container/constant/constant";
import api from "../../core/utilities/httpProvider";

export const forgotPasswordService = (data) => {
  return api.post(FORGOT_PASSWORD, {
    email: data.payload.email,
  });
};
