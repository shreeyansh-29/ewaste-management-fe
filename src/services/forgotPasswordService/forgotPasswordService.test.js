/* eslint-disable no-undef */

import {forgotPasswordService} from "./forgotPasswordService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing forgotPasswordService", () => {
  const data = {payload: {email: "shreeyansh.singh@nineleaps.com"}};
  const Service = forgotPasswordService;
  const wrapper = shallow(<Service {...data} />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
