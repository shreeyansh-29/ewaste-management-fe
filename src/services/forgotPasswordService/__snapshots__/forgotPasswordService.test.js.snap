// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing forgotPasswordService 1`] = `<Component />`;
